notebooks/book2/29/hmm_casino_training.ipynb

```diff
@@ -0,0 +1,1002 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    from graphviz import Digraph
</pre><p>here you should put <code>pip </code> install graphviz in try except block.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>